kotlin-sdk/app/src/main/res/layout/activity_todo.xml

```diff
@@ -4,10 +4,10 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
```

Comment:
Should we update this file name to `activity_item.xml` + rename when we reference it?

Reply:
Yes, good catch - prior to this I replaced 'todo' with 'item' for the in-code usages, but hadn't renamed the files + file references. I've done that now.